daemon/container_operations.go

```diff
@@ -485,6 +486,16 @@ func (daemon *Daemon) updateNetworkConfig(container *container.Container, idOrNa
 		if endpointConfig != nil && len(endpointConfig.Aliases) > 0 {
 			return nil, runconfig.ErrUnsupportedNetworkAndAlias
 		}
+	} else {
+		addShortID := true
+		for _, alias := range endpointConfig.Aliases {
+			if alias == stringid.TruncateID(container.ID) {
```

Comment:
minor nit, but perhaps `stringid.TruncateID(container.ID)` should be put in a variable instead of truncating it inside the loop